host/src/lib.rs

```diff
@@ -665,25 +665,4 @@ impl<'stack, C: Controller, P: PacketPool> Stack<'stack, C, P> {
     pub fn log_status(&self, verbose: bool) {
         self.host.log_status(verbose);
     }
-
-    #[cfg(feature = "security")]
```

Comment:
Those functions should not be removed, the BLE host needs to provide a way for retrieving and loading bond information which is saved on the user-side

Reply:
I can certainly add that back in. In the future I want to explore alternative ways of handling bonding storage with the goal of removing the `BOND_COUNT` generic argument, but until then i will restore the original capability

Reply:
Can the `bondable` be set automatically according to the advertising type? Or could you add an example for creating bondable connection and saving & loading the bond information?

Reply:
Bondable isn't detectable by advetisement type. It is a capability where a device should only be bondable if it is possible to store bonding info in NVM. And it should only be stored/reused if both central and peripheral supports bonding.

In the current implementation the central handling is wrong since it will try to enable encryption without pairing if it has successfully paired previously, even if the peripheral device doesnt support bonding (and thus wont store the LTK). 

I think it would make sense to have a (stack-)global default that can also be overridden per connection. This is also what Zephyr does.

I can also add an example.

Reply:
thanks for the explanation!